### DISCUSSION TITLE
Thank you for this, Directus!

### DISCUSSION THREAD

**Discussion by notflip** [directus-labs/agency-os#12] [General]
This is just wonderful, thanks for taking the time to build this.

**Comment by bryantgillespie** [comment]
You're very welcome. We appreciate the feedback!

**Reply by Heartnett** [reply]
@bryantgillespie will there be a version of the agency-os template with websockets support baked-in as the default instead of REST?

**Reply by bryantgillespie** [reply]
As a default for fetching data? No – I don't think so.

It might make sense within the client portal layer to have web sockets for things like conversations or notifications.

**Reply by Heartnett** [reply]
@bryantgillespie thanks, makes sense. 👍
